crates/pixi_build_frontend/src/tool/cache.rs

```diff
@@ -266,15 +268,24 @@ impl ToolContext {
     }
 }
 
-/// A [`ToolCache`] maintains a cache of environments for build tools.
+/// A record that is either pending or has been fetched.
```

Comment:
```suggestion
/// A record that is either pending or has been fetched.
```

Maybe something other than record?